rosa_architecture/rosa-aws-amis.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-aws-ami"]
+= AWS AMIs for {product-title}
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-aws-ami
+
+toc::[]
+
+The following list contains the Amazon Machine Images (AMIs) for {product-title} clusters. For more information, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html#ami-using[Amazon Machine Images (AMI)] in the AWS documentation.
+
+You must use the image version that is the same as the AMI version when you installed your cluster. For example, if you installed the cluster using 4.14, but your cluster is on OpenShift version 4.16, you must still allow for AMIs that use the `rhcos-414-...` because new instances use same AMI version.
+
+include::modules/rosa-classic-aws-rhcos-ami.adoc[leveloffset=+1]
+//include::modules/rosa-hcp-aws-rhcos-ami.adoc[leveloffset=+1]
```

Comment:
Is there a reason the HCP module is commented out?

Reply:
When we generated the ROSA AMIs, @iamkirkbater was pretty sure that the HCP ones were different, and we were going to update those when we could confirm.

Reply:
Sounds good, just wanted to verify.

Reply:
So I just confirmed - HCP uses the OCP AMI ID the same as OpenShift Classic, where ROSA uses a separate AMI ID for billing through AWS.

I've also asked about the process for upgrading HyperShift nodes and whether they use the same AMI ID even with future upgrades, but I haven't gotten a clear answer yet.

Reply:
@michaelryanmcneill and @iamkirkbater to make sure I am understanding clearly, do I need to structure this section as the ROSA with HCP AMIs table? https://docs.openshift.com/container-platform/4.15/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-user-infra-rhcos-ami_installing-restricted-networks-aws